Public/Src/Cache/ContentStore/Distributed/Stores/DistributedContentCopier.cs

```diff
@@ -58,6 +58,12 @@ public class DistributedContentCopier<T> : StartupShutdownSlimBase, IDistributed
         /// <nodoc />
         public int CurrentIoGateCount => _ioGate.CurrentCount;
 
+        /// <summary>
+        /// Controls the maximum total number of copy retry attempts
+        /// </summary>
+        public const int MaxRetryCount = 32;

```

Comment:
nit: formatting. Could you please remove an extra blank line after this const declaration?